app/ide-desktop/client/tests/localWorkflow.spec.ts

```diff
@@ -106,6 +119,9 @@ electronTest('Local Workflow', async ({ page, app, projectsDir }) => {
   expect(projectFiles).toContain('images')
   const images = await fs.readdir(pathModule.join(PROJECT_PATH, 'images'))
   expect(images).toContain('image.png')
+  await expect(page.locator('.GraphEditor .GraphNode.pending')).toHaveCount(0)
```

Comment:
This is a strange check - we don't touch code after last such check - if it's needed, please add info why.